<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDlg</class>
 <widget class="QDialog" name="ReportDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stampa</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Report:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="reportComboBox">
       <item>
        <property name="text">
         <string>Listino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ordine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inventario</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fornitore:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fornitoreComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="listinoGroupBox">
     <property name="title">
      <string>Listino</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="listinoPrintCurrentDate">
        <property name="text">
         <string>Stampa listino con data arrivo odierna</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="listinoPrintAll">
        <property name="text">
         <string>Stampa listino completo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ordineGroupBox">
     <property name="title">
      <string>Ordine</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="inventarioGroupBox">
     <property name="title">
      <string>Inventario</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pdfCheckBox">
     <property name="text">
      <string>Stampa su pdf</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="configButton">
       <property name="text">
        <string>C&amp;onfigura</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printButton">
       <property name="text">
        <string>&amp;Stampa</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ReportDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printButton</sender>
   <signal>clicked()</signal>
   <receiver>ReportDlg</receiver>
   <slot>print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configButton</sender>
   <signal>clicked()</signal>
   <receiver>ReportDlg</receiver>
   <slot>launchConfigDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reportComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>ReportDlg</receiver>
   <slot>showOptions(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>print()</slot>
  <slot>launchConfigDlg()</slot>
  <slot>showOptions(QString)</slot>
 </slots>
</ui>
